### ISSUE TITLE
chinese map source

### ISSUE THREAD

**Issue by kapdon** [UnclePez/dfho#2]
in case you didn't know this existed. https://df.qq.com/cp/a20240729directory/

started some process of making this for personal use then found you repo.
not sure if you are open to prs/contributors at this stage but lmk if you need any help.

**Comment by UnclePez** [comment]
Yes I've seen that site, I think it's official but isn't quite optimized yet. I'm mostly making my own for fun and to learn coding and web hosting. If you want to help with locations or other info feel free to msg me (UnclePez on Discord), but this repo will continue being a personal project for now
